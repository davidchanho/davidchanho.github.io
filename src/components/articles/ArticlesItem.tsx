import React from 'react'
import { ListGroupItem } from 'react-bootstrap'

function ArticlesItem() {
    return (
        <ListGroupItem>
            
        </ListGroupItem>
    )
}

export default ArticlesItem
