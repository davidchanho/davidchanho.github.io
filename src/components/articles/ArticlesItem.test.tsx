import React from "react";
import renderer from "react-test-renderer";
import ArticlesItem from "./ArticlesItem";

describe("ArticlesItem", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ArticlesItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
