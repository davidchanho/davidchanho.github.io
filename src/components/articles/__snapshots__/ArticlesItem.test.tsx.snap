// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticlesItem renders correctly 1`] = `
<div
  className="list-group-item"
  disabled={false}
  onClick={[Function]}
/>
`;
